auth0/src/test/java/com/auth0/android/result/CredentialsTest.kt

```diff
@@ -35,4 +41,22 @@ public class CredentialsTest {
         credentials.recoveryCode = "recoveryCode"
         MatcherAssert.assertThat(credentials.recoveryCode, Matchers.`is`("recoveryCode"))
     }
+
+    @Test
+    public fun shouldGetUser() {
+        val date = Date()
+        val credentials: Credentials =
+            CredentialsMock(idToken, "accessToken", "type", "refreshToken", date, "scope")
+        MatcherAssert.assertThat(credentials.user.getId(), Matchers.`is`("auth0|123456"))
+    }
+
+    @Test
+    public fun shouldNotSerializeUser() {
```

Comment:
Should we also test the deserialized values?

Reply:
What would we be testing?